pallets/capacity/src/benchmarking.rs

```diff
@@ -11,6 +11,7 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::RuntimeEvent) {
 	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
 }
 
+#[allow(clippy::expect_used)]
```

Comment:
Since this is benchmarking, should this be for the entire file/module?

Reply:
I think we should keep the exceptions scope as limited as possible. If the we need to expand the exception to more than one location, it might make sense to make it file/module scope.